Doberman/AlarmNode.py

```diff
@@ -51,21 +52,26 @@ def log_alarm(self, msg, ts=None):
         """
         Let the outside world know that something is going on
         """
-        if not self.is_silent or self.pipeline.silenced_at_level < self.base_level:
+        # Only send message if pipeline is silenced at base_level or above, 
+        # or if it is silenced at level -1 (universal)
+        if not self.is_silent or -1 < self.pipeline.silenced_at_level < self.base_level:
             self.logger.debug(msg)
```

Comment:
Not related, but shouldn't we make this a logger.error() and line 76 a logger.debug() or nothing?

Reply:
Do we really want an error every time an alarm is not sent because pipeline is silenced? It would mean 1 entry in the logs per second for most alarms where the sensor is read out once per second